airbyte-webapp/src/views/Connector/ConnectorForm/connectorFormContext.tsx

```diff
@@ -63,7 +63,7 @@ export const ConnectorFormContextProvider: React.FC<React.PropsWithChildren<Conn
 
   const ctx = useMemo<ConnectorFormContext>(() => {
     const unfinishedFlows = widgetsInfo["_common.unfinishedFlows"] ?? {};
-    return {
+    const context: ConnectorFormContext = {
```

Comment:
:information_source: Assigning this object to a variable that's explicitally types, will cause TypeScript to now properly check if the object has keys that it shouldn't have.